src/App.vue

```diff
@@ -1,10 +1,75 @@
-html,
-body {
+<template>
+  <section class="todoapp">
+    <todo-form v-on:@add="addNewItem"></todo-form>
```

Comment:
```suggestion
    <todo-form v-on:@add="addNewItem" />
```
이건 개인 취향일 수 있는데요.
vue에서 컴포넌트는 단일 태그로도 사용할 수 있답니다!
컴포넌트 안의 내용이 없다면 단일 태그를 사용해보는 건 어떨까요?

Reply:
훨씬 간결하군요! 👍